teaser/examples/e11_export_besmod_models.py

```diff
@@ -0,0 +1,195 @@
+# # Example 11: Export Modelica models for BESMod library using TEASER API
+
+# This module demonstrates how to export building models from a TEASER project
+# to ready-to-run simulation models for the Modelica BESMod library.
+# You can execute this example using
+# [jupyter-notebook](https://mybinder.org/v2/gh/RWTH-EBC/TEASER/master?labpath=docs%2Fjupyter_notebooks)
+
+import teaser.examples.e1_generate_archetype as e1
+import teaser.logic.utilities as utilities
+import os
+
+
+def example_export_besmod(save_dir_path=None):
+    """This function demonstrates the export to Modelica library BESMod using
+    the API function of TEASER"""
+
+    # In e1_generate_archetype we created a Project with three archetype
+    # buildings to get this Project we rerun this example
+
+    prj = e1.example_generate_archetype()
+
+    # Configure project settings to ensure compatibility with BESMod. The BESMod
+    # library uses the AixLib.ThermalZones.ReducedOrder.ThermalZone.ThermalZone model
+    # with 4 elements for the demand building model. Set these parameters in the project:
+    prj.used_library_calc = 'AixLib'
+    prj.number_of_elements_calc = 4
+
+    # BESMod allows building models to be included in predefined example energy systems:
+    examples = [
+        "TEASERHeatLoadCalculation",  # Ideal electric heater for heat load calculations
+        "HeatPumpMonoenergetic",  # Heat pump with radiators, buffer storage, and PV
+        "GasBoilerBuildingOnly"  # Gas boiler with radiators
+    ]
+
+    # For hydraulic systems, specify the nominal hydraulic supply temperature.
```

Comment:
For the hydraulic systems, you have to specify a nominal supply temperature for heat transfer, e.g. in radiators.